frontend/src/Components/Logo/components/UkaLogo.tsx

```diff
@@ -0,0 +1,63 @@
+import { LogoPalette, LogoProps } from '~/Components/Logo/Logo';
+import { COLORS } from '~/types';
+
+type UkaLogoProps = {
+  color: LogoProps['color'];
+  size: LogoProps['size'];
+};
+
+export function UkaLogo({ color, size }: UkaLogoProps) {
```

Comment:
Foretrekker å bruke riktig bokstavstørrelse her, så det blir UKA og ikke Uka.

```suggestion
export function UKALogo({ color, size }: Props) {
```